index.html

```diff
@@ -10,15 +10,51 @@
     crossorigin="anonymous"
     href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"
     integrity="sha512-1PKOgIY59xJ8Co8+NE6FZ+LOAZKjy+KY8iq0G4B3CyeY6wYHN3yt9PW0XpSriVlkMXe40PTKnXrLnZ9+fkDaog==">
+  <link href="https://fonts.googleapis.com/css2?family=Proza+Libre&display=swap" rel="stylesheet">
   <link rel="stylesheet" href="css/reset.css">
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
 </head>
 <body>
-  <header>
-  </header>
-  <main>
-  </main>
+  <div class="center">
+
+    <header class="headers">
+      <nav class="container row headers full-width header-position">
+        <h1 class="headers main-header headings">Code Journal</h1>
+      </nav>
+    </header>
+
+    <main class="container full-width">
+      <div class="entry-form-page" data-view="entry-form">
+        <form class="row center">
+          <h1 class="new-entry-header column-full headings">New Entry</h1>
+
+          <div class="column-half image-container">
+            <img class="preview" id="image" src="images/placeholder-image-square.jpg" alt="placeholder-image">
+          </div>
+          <div class="column-half">
+            <div class="row">
+              <label class="full-width headings" for="title">Title</label>
+              <input class="full-width text-input title" type="text" required="required">
+            </div>
+            <div class="row">
+              <label class="full-width headings" for="photourl">Photo URL</label>
+              <input class="full-width text-input photourl" type="url" required="required">
+            </div>
+          </div>
+
+          <div class="column-full row">
```

Comment:
What is the purpose of the `row` class here? I don't see any `column` class elements as children of this. Also, no one element should ever be both a `column` and a `row`. If you need a `row` in a `column`, you should make it a child element of the `column`.